sky/services/activity/activity.mojom

```diff
@@ -67,6 +69,10 @@ interface Activity {
   GetCacheDir() => (string path);
 };
 
+interface UpdateService {
+  NotifyUpdateCheckComplete();
```

Comment:
This provides a way for Dart to notify Java that it's done. I'm thinking I should use native functions instead, so it would go from Dart -> C++ -> Java. But I wrote it this way first, so I'll see what you think.